From eb92134aa9f9274dbc24496e9012885d2cbbe94a Mon Sep 17 00:00:00 2001
From: Carson Black <uhhadd@gmail.com>
Date: Wed, 23 Sep 2020 14:57:45 -0400
Subject: [PATCH] [gtk3]: Make scrollbars fixed size

This better matches their behaviour in the QStyle and fixes a bug.

BUG: 414763
---
 src/gtk3/widgets/_scrollbar.scss | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/gtk3/widgets/_scrollbar.scss b/src/gtk3/widgets/_scrollbar.scss
index 6f9237c..7036e58 100644
--- a/src/gtk3/widgets/_scrollbar.scss
+++ b/src/gtk3/widgets/_scrollbar.scss
@@ -66,7 +66,7 @@ scrollbar {
   trough {
     //margin: 0px;
     transition-duration: 0.1s;
-    min-width: 3px;
+    min-width: 6px;
     min-height: 14px;
     border-radius: 8px;
     background-color: transparent;
@@ -87,7 +87,7 @@ scrollbar {
     //margin: 0px;
     transition-duration: 0.1s;
     min-width: 14px;
-    min-height: 3px;
+    min-height: 6px;
     border-radius: 8px;
     background-color: transparent;
   }
@@ -106,7 +106,7 @@ scrollbar {
   slider {
     //margin: 0px;
     transition-duration: 0.1s;
-    min-width: 3px;
+    min-width: 6px;
     min-height: 30px;
     border-radius: 8px;
     background-clip: padding-box;
-- 
GitLab

